js/script.js

```diff
@@ -282,7 +282,9 @@ $(function(){
 			$element.find('.avatar-name-wrapper').each(function() {
 				var element = $(this);
 				var avatar = element.find('.avatar');
+				avatar.attr('tabindex', 0)
```

Comment:
Why not put it in the template? https://github.com/nextcloud/activity/blob/master/js/templates/userLocal.handlebars